data/map.txt

```diff
@@ -31454,6 +35775,24 @@ planet Shroud
 		threshold 7000
 		fleet "Large Republic" 14
 
+planet Siedi
+	landscape land/forest5
+	description `The gravitational pull of the nearby gas giant resulted in the large, bulky, mountainous surface of this planet. Tectonic shifts are very common, and the surface is marked with the wear and tear of massive tremors. Beneath the cracks, however, lies a rich source of materials that the Vi quickly capitalized on for their nearby worlds.`
+	spaceport `The spaceport is haphazardly built inside the side of a smaller mountain, and every now and then you can feel a very faint rumbling from below. You're not sure if it is beneath you or somewhere off in the distance. The area around the spaceport continues to be under construction, but it is more likely because the Vi are mining for materials here rather than making improvements to the structure itself.`
```

Comment:
```suggestion
	spaceport `The spaceport is haphazardly built into the side of a smaller mountain, and every now and then you can feel a very faint rumbling from below. You're not sure if it is beneath you or somewhere off in the distance. Construction work around the spaceport is ongoing, but it is more likely because the Vi are finding new places to mine for materials rather than making improvements to the spaceport itself.`
```